File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        sum = 0
        max = 0
        for i in range(0,len(gain)):
            sum = sum+gain[i]
            if(sum>max):
                max = sum
        return max

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.js
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function (gain) {
    let sum = 0;
    let max = 0;
    for (let i = 0; i < gain.length; i++) {
        sum = sum + gain[i]
        if (sum > max) {
            max = sum
        }
    }
    return max
};